gatsby/assets/ds/scss/base/_variables.scss

```diff
@@ -11,35 +11,35 @@ $theme-white:                 $white;                              // Pure white
 $black:                       #000;                              // Pure black color
 $theme-black:                 $black;                              // Pure black color
 
-$theme-blue-lighten:          #3375ae;
-$theme-blue-light:            #337dbd;
-$theme-blue:                  #2362a2;
-$theme-blue-dark:             #254e80;
+$theme-blue-lighten:          #3375ae !default;
+$theme-blue-light:            #337dbd !default;
+$theme-blue:                  #2362a2 !default;
+$theme-blue-dark:             #254e80 !default;
 
-$theme-yellow:                #ecae1a;
-$theme-red:                   #e04857;
-$theme-gray-dark:             #3b3b3b;
-$theme-gray:                  #707070;
-$theme-gray-lighten:          #e1e1e1;
-$theme-gray-light:            #f5f5f5;
+$theme-yellow:                #ecae1a !default;
+$theme-red:                   #e04857 !default;
+$theme-gray-dark:             #3b3b3b !default;
+$theme-gray:                  #707070 !default;
+$theme-gray-lighten:          #e1e1e1 !default;
+$theme-gray-light:            #f5f5f5 !default;
 
 
 // Layout
 $border-radius:               3px !default;
 $enable-shadows:              true !default;
 
-$theme-bg:                    $theme-gray-light;
+$theme-bg:                    $theme-gray-light !default;
 $theme-font-size-base-raw:    14;
 $theme-font-size-base:        14px !default;
 
-$theme-class-def:             'pvs-theme';
-$theme-class:                 '.#{$theme-class-def}';
-$theme-path:                  '/ds';
-$theme-text-color:            $theme-gray;
-$theme-letter-spacing:        .075em;
-$theme-headings-color:        $theme-gray-dark;
+$theme-class-def:             'pvs-theme' !default;
+$theme-class:                 '.#{$theme-class-def}' !default;
+$theme-path:                  '/ds' !default;
```

Comment:
tady je default '', vyhoď to prosím taky do toho designsystem.scss, ať máme ve složce ds opravdu původní design systém

Reply:
@vitkolos pokud tam není !default, tak mi to nešlo přepsat pak tu variable, mám teda nechat _variables.scss originální a půjdou měnit pouze ty variables u kterých mají !default přidáno oni?

Reply:
@vitkolos Nejde pak přepsat ani `$theme-path` tám pádem se nenačítají ikony atd..

Reply:
no to je fakt, kašli na to

Reply:
původní design systém tam mít budeme až na ty default 👍